src/scripts/main.js

```diff
@@ -1,3 +1,18 @@
 'use strict';
 
-console.log('hello mate academy!');
+const menu = () => {
+  const BURGER = document.querySelector('.header__button');
```

Comment:
это не следует называть в верхнем регистре, это просто переменная
```suggestion
  const burger = document.querySelector('.header__button');
```